packages/metro/src/Bundler/index.js

```diff
@@ -155,7 +156,7 @@ class Bundler {
       transformModuleHash,
     ];
 
-    this._getModuleId = createModuleIdFactory();
+    this._getModuleId = (opts.createModuleIdFactory || createModuleIdFactory)();
```

Comment:
You can safely delete these 3 occurrences here, I forgot to remove them when deprecating the old bundler 😄